integration_test/buckets.tf.template

```diff
@@ -6,6 +6,25 @@ resource "aws_s3_bucket" "internal" {
     Application = "ORCA"
   }
 }
+# todo: Either commit to us-west-2 and remove user's ability to change it, or make 797873946194 variable with instructions.
```

Comment:
I eel you can make us-west-2 as the region since that is unlikely users will move there data out of the region based on current NASA contracts. I would also make the account number a variable since we wouldn't want to expose that in the git repo. It would probably be easier to dynamically pull both of these values from terraform if we can like we do here https://github.com/nasa/cumulus-orca/blob/8625e9ebb3e5f7a437a9f5053addbd1384a1f1ea/modules/secretsmanager/main.tf#L14

Reply:
For first issue:
Removing the account id is beyond the scope of this PR. Created https://bugs.earthdata.nasa.gov/browse/ORCA-704
I will not be attempting to complete it before I leave.

Reply:
Please remember to split multiple issues into their own cards.

Reply:
Card created for the first issue: https://bugs.earthdata.nasa.gov/browse/ORCA-704

Reply:
For the second issue, I did what I could, but the dr buckets also need this value. We already expose that value, so this is not a new issue that blocks the PR. Created https://bugs.earthdata.nasa.gov/browse/ORCA-705